include/circt/Dialect/FIRRTL/Passes.h

```diff
@@ -142,7 +143,7 @@ std::unique_ptr<mlir::Pass> createRandomizeRegisterInitPass();
 std::unique_ptr<mlir::Pass> createLowerXMRPass();
 
 std::unique_ptr<mlir::Pass>
-createResolveTracesPass(StringRef outputAnnotationFilename = "");
+createResolveTracesPass(mlir::StringRef outputAnnotationFilename = "");
```

Comment:
Surprising that this is necessary. 😕  Non-blocking.

Reply:
May not be, to be honest, but clang-tidy was complaining and I think it's right.  Similarly use of incomplete types w/`std::unique_ptr` that it catches (maybe what we have compiles because other things are included first)?  I didn't tackle those since they're already present and pretty sure last time 'round we ignored them too.

Anyway this seemed good since it's the only non-scoped reference to `StringRef` in the file, so for consistency reasons if nothing else :+1:.